### PR TITLE
Add profile picture for v1.1

### PR DESCRIPTION
**Pull request by russelltankaimin**

### CODE REVIEW

File: docs/team/russelltankaimin.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: page
+title: Tan Kai Min, Russell's Project Portfolio Page
+---
+
+### Project: EduMate
+
+EduMate is a desktop address book application used to help NUS students maintain both their social and academic life by lowering the barriers to meet up and also make new friends within modules and school. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
```
Comment (mynameizzhafeez) [inline docs/team/russelltankaimin.md:9]: We need to standardise this eventually.

**Review by mynameizzhafeez — APPROVED** [review]
LGTM.

File: docs/team/russelltankaimin.md (hunk 1 above)
Comment (russelltankaimin) [reply docs/team/russelltankaimin.md:9]: I used the one from user guide.

Comment (mynameizzhafeez) [reply docs/team/russelltankaimin.md:9]: I did the same, but what do we want it to be in the end?

**Pull request merged**